variables/_svgs.pcss

```diff
@@ -299,6 +323,18 @@
 
 @svg-load no-map url('../icons/no-map.svg') {}
 
+/* -----------------------------------------------------------------------------
+ * SVG - Phone
+ * ----------------------------------------------------------------------------- */
+
+@svg-load phone url('../icons/phone.svg') {
+
+	path {
+		stroke: var(--color-icon-active);
+	}
+
```

Comment:
```suggestion
```

Reply:
The rest of the variables have this trailing newline. If I remove the line will be inconsistent.

Reply:
No need to change. Keep it consistent

Reply:
I took another look, the file doesnt have the last spacing. So I am pushign the suggestion from @paulmskim